python/py_opt_control/min_time_bvp.py

```diff
@@ -171,18 +172,18 @@ def switch_states(p0, v0, a0, t, j):
 
     return a, v, p
 
-def sample_min_time_bvp(p0, v0, a0, t, j, dt):
+def sample(p0, v0, a0, t, j, st):
     """
     Given an initial state and an input switching sequence, compute the full
-    state at sampled times with resolution dt.
+    state at a set of explicitly specified sample times. 
 
     Inputs:
         p0, initial position,     shape=(N,)
         v0, initial velocity,     shape=(N,)
         a0, initial acceleration, shape=(N,)
         t,  switch times, shape=(N,M)
         j,  jerk,         shape=(N,M)
-        dt, time steps
+        st, samplle_time, shape=(K,)
     Outputs:
         st, times,        shape=(N,K)
```

Comment:
This function no longer needs to return st -- it can be removed from both the comments and the return statement.